src/analyze/gui/parser.cpp

```diff
@@ -644,6 +644,11 @@ void Parser::parse(const QString& path, const QString& diffBase)
 
 void Parser::parseImpl(const QString& path, const QString& diffBase, const FilterParameters& filterParameters)
 {
+    using std::chrono::steady_clock;
```

Comment:
I personally find it easier to use QElapsedTimer for this purpose, could you please leverage that one?

Reply:
Yep. I'm still learning some parts of the Qt ecosystem. I'll read up and switch it.